src/views/settings/Auth/const.js

```diff
@@ -47,7 +47,7 @@ export function getAuthItems() {
       description: ''
     },
     {
-      name: 'OAUTH2',
+      name: 'OAuth2',
       logo: require('@/assets/img/auth/oauth2_logo.png'),
       title: i18n.t('OAuth2'),
       authKey: 'AUTH_OAUTH2',
```

Comment:
No major issues noted in the provided code snippet. However, to include some minor suggestions:

- The "description" property is not used after adding its type and properties section.
- Instead of creating two separate functions, you might want to refactor into one that handles both OAuth1 and OAuth2.

Here's a summary:
- No significant code differences found initially.
  
Suggested Changes (as per comments):
- Add a comment on the first `description` line indicating its absence since it does not seem necessary here.
- Refactor the entire codeblock to handle both types with single function.